lib/Clam/CfgBuilder.cc

```diff
@@ -2537,6 +2626,58 @@ void CrabIntraBlockBuilder::LoadFromSingletonMem(LoadInst &I, var_t lhs_v,
   }
 }
 
+// -- dereference a ptr with bound checks
+// In addition, crab IR we insert the following assertions:
+// assert(offset >= 0)
+// assert(obj size > offset)
+// assert(obj size >= offset + size_of(value))
+// where offset indicates the offset of current pointer
+// the size means the total size of allcation
+// sizeof(value) gets the bit-width of the element
+void CrabIntraBlockBuilder::doDerefCheck(Instruction &I) {
```

Comment:
call it `doDerefBndCheck` to be consistent with the rest.

Reply:
Done!